datasets/cmu_mocap/README.md

```diff
@@ -0,0 +1,169 @@
+---
+annotations_creators:
+- expert-generated
+language_creators:
+- expert-generated
+languages:
+- en
+licenses:
+- unknown
+multilinguality:
+- monolingual
+pretty_name: CMU MoCap
+size_categories:
+- 1K<n<10K
+source_datasets:
+- original
+task_categories:
+- other
+task_ids: []
+paperswithcode_id: mocap
+---
+
+# Dataset Card for CMU MoCap 
+
+## Table of Contents
+- [Dataset Description](#dataset-description)
+  - [Dataset Summary](#dataset-summary)
+  - [Supported Tasks](#supported-tasks-and-leaderboards)
+  - [Languages](#languages)
+- [Dataset Structure](#dataset-structure)
+  - [Data Instances](#data-instances)
+  - [Data Fields](#data-instances)
+  - [Data Splits](#data-instances)
+- [Dataset Creation](#dataset-creation)
+  - [Curation Rationale](#curation-rationale)
+  - [Source Data](#source-data)
+  - [Annotations](#annotations)
+  - [Personal and Sensitive Information](#personal-and-sensitive-information)
+- [Considerations for Using the Data](#considerations-for-using-the-data)
+  - [Social Impact of Dataset](#social-impact-of-dataset)
+  - [Discussion of Biases](#discussion-of-biases)
+  - [Other Known Limitations](#other-known-limitations)
+- [Additional Information](#additional-information)
+  - [Dataset Curators](#dataset-curators)
+  - [Licensing Information](#licensing-information)
+  - [Citation Information](#citation-information)
+
+## Dataset Description
+
+- **Homepage:** [CMU MoCap homepage](http://mocap.cs.cmu.edu/)
+- **Repository:** [More Information Needed]
+- **Paper:** [More Information Needed]
+- **Leaderboard:** [More Information Needed]
+- **Point of Contact:** [CMU MoCap Contact](mailto:jkh+mocap@cs.cmu.edu)
+
+### Dataset Summary
+
+The CMU Graphics Lab Motion Capture Database is a dataset containing motion capture recordings of people performing different actions such as walking, jumping, dancing etc. The actions have been performed by over 140 subjects. There are a total of 2605 trials in 6 categories and 23 subcategories.
```

Comment:
According to the generated infos file, the number of examples is 112, so we should update the numbers in this section accordingly if that's correct. 

I would expect 144 [subjects](http://mocap.cs.cmu.edu/subjects.php)/rows in the dataset. Is the metadata file missing some subjects? If yes, is there something we can do to get them?

Reply:
Yes. The no.of examples are 112, I'll update this section.
The subject numbers go upto 144 (http://mocap.cs.cmu.edu/subjects.php) but some of the subject numbers are missing, so total count of examples come up to 112 itself.

Reply:
Maybe we can add a note to let the users know that some examples are not present on the website.